docs/examples/zeroable/index.html

```diff
@@ -62,7 +62,7 @@ <h3>zeroable</h3>
 <br>
 <div style="display: flex; justify-content: space-between;">
     <a href="https://cairo-by-example.com/examples/enums/">prev (enums)</a>
-    <a href="https://cairo-by-example.com/examples/snapshots/">next (snapshots)</a>
+    <a href="https://cairo-by-example.com/examples/spans/">next (spans)</a>
```

Comment:
why did you removed this one?

Reply:
That's weird. I don't know why Hugo removed this

Reply:
Oh. I know. It's because the "next example" for _zeroable_ changed from _snapshots_ to _spans_